desktop-src/direct3d12/recording-command-lists-and-bundles.md

```diff
@@ -217,6 +217,78 @@ Applications can expect to see DXGI\_DEVICE\_REMOVED errors in the following API
 -   [**IDXGISwapChain1::Present1**](https://docs.microsoft.com/windows/desktop/api/dxgi1_2/nf-dxgi1_2-idxgiswapchain1-present1)
 -   [**GetDeviceRemovedReason**](/windows/desktop/api/d3d12/nf-d3d12-id3d12device-getdeviceremovedreason)
 
+## Command list API Restrictions
+
+Some command list APIs can only be called on certain types of command lists. The table below shows which command list APIs are valid to call on each type of command list. It also shows which APIs are valid to call in a [**D3D12 render pass**](/windows/desktop/direct3d12/direct3d-12-render-passes). 
```

Comment:
Is there an easy way to check if this link works correctly?

Reply:
Normally, your branch should show up in the drop-down here:
https://review.docs.microsoft.com/en-us/windows/win32/direct3d12/recording-command-lists-and-bundles

however I don't see it. Maybe, because it's a fork and not a private branch off of master. @stevewhims do you know?

Reply:
![image](https://user-images.githubusercontent.com/8118775/72660873-746f2d00-3988-11ea-81a7-2bb1973e62c1.png)

Reply:
Normally, it should show up here: 
https://review.docs.microsoft.com/en-us/windows/win32/direct3d12/recording-command-lists-and-bundles

![image](https://user-images.githubusercontent.com/8118775/72660886-b9935f00-3988-11ea-851b-a96d296d5a4d.png)

But I don't see it. This might be because this branch is based off a fork not a branch off the normal mainline... @stevewhims do you know?

Reply:
This PR uses the public repo workflow (which we recommend for non-Microsoft folks), so I don't know how (or whether) one can preview one's changes. If you go with the private repo workflow (which we recommend for Microsoft employees and it's doc'd here: https://review.docs.microsoft.com/windows-authoring-guide/quickstart?branch=master), then you'll be able to go to review.docs.microsoft.com, find your topic, and select your branch. But that's the right format for links (starting with /windows). You can just prepend "https://docs.microsoft.com/" to any link in that format to confirm that it's good (or use the review site, as we said). However, "/windows/desktop" will be redirected to "/windows/win32" so you might as well just author the correct url in new links.